src/Admin/AdminInterface.php

```diff
@@ -678,7 +683,11 @@ public function getListMode();
 //    public function showMosaicButton(bool $isShown): void;
 
     /*
-     * Checks if a filter type is set to a default valu
+     * Checks if a filter type is set to a default value
+     *
+     * @param string $name
```

Comment:
These docblocks are not bringing any useful information, since real type declarations are already added.